src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -57,41 +97,135 @@ public function register_tab() {
 
 		new \Tribe__Settings_Tab( static::$slug, esc_html__( 'Payments', 'event-tickets' ), $tab_settings );
 	}
-
-
+	
 	/**
-	 * Gets the top level settings for Tickets Commerce.
+	 * Returns the settings item for the section menu at the top of the Payments settings tab.
 	 *
-	 * @since 5.2.0
+	 * @since TBD
 	 *
 	 * @return array[]
 	 */
-	public function get_top_level_settings() {
-
-		$plus_link    = sprintf(
-			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
-			esc_html__( 'Event Tickets Plus', 'event-tickets' )
-		);
-		$plus_link_2  = sprintf(
-			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
-			esc_html__( 'Check it out!', 'event-tickets' )
+	public function get_section_menu() {
+		
+		$gateways = tribe( Manager::class )->get_gateways();
+		$selected_section = tribe_get_request_var( self::$key_current_section_get_var );
+		$menu_html = '<div class="tec-tickets__admin-settings-tickets-commerce-section-menu">';
```

Comment:
I'm curious why we're not getting this from admin views, do you think you could possibly move this to templates given that we're managing HTML?

Reply:
I'm working on moving those into templates as we speak. G mentioned that, too.